modules/openapi-generator/src/main/resources/JavaSpring/apiUtil.mustache

```diff
@@ -27,7 +28,7 @@ public class ApiUtil {
 {{/reactive}}
 {{#reactive}}
     public static Mono<Void> getExampleResponse(ServerWebExchange exchange, String example) {
-        return exchange.getResponse().writeWith(Mono.just(new DefaultDataBufferFactory().wrap(example.getBytes())));
+        return exchange.getResponse().writeWith(Mono.just(new DefaultDataBufferFactory().wrap(example.getBytes(StandardCharsets.UTF_8))));
     }
 {{/reactive}}
 }
```

Comment:
it would be nice to take advantage of  this PR to fix the missing empty new line on this file

Reply:
Missing newline fixed, thanks!